tests/rados/stretch_cluster.py

```diff
@@ -59,6 +59,65 @@ def run(ceph_cluster, **kw):
         )
         return 1
 
+    if config.get("verify_forced_recovery"):
+        log.info("Verifying forced recovery and healthy in stretch environment")
+
+        pool_name = "stretch_pool_recovery"
+        if not rados_obj.create_pool(pool_name=pool_name, pg_num=16):
+            log.error("Failed to create the replicated Pool")
+            return 1
+
+        # getting the acting set for the created pool
+        acting_pg_set = rados_obj.get_pg_acting_set(pool_name=pool_name)
+
+        log.info(
+            f"Killing 2 OSD's from acting set : {acting_pg_set} to verify recovery"
+        )
+        stop_osds = [acting_pg_set.pop() for _ in range(2)]
+        for osd_id in stop_osds:
+            if not rados_obj.change_osd_state(action="stop", target=osd_id):
+                log.error(f"Unable to stop the OSD : {osd_id}")
+                return 1
+
+        # Sleeping for 25 seconds ( "osd_heartbeat_grace": "20" ) for osd's to be marked down
+        time.sleep(25)
+
+        log.info("Stopped 2 OSD's from acting set, starting to wait for recovery")
+        rados_obj.change_recover_threads(config=config, action="set")
+
+        if not rados_obj.bench_write(pool_name=pool_name, **config):
+            log.error("Failed to write objects into the Pool")
+            return 1
+
+        log.debug("Triggering forced recovery in stretch mode")
+        if not stretch_mode_force_recovery(node=cephadm):
+            log.error("Failed to trigger forced recovery")
+            return 1
+
+        log.debug("Starting the stopped OSD's")
+        for osd_id in stop_osds:
+            if not rados_obj.change_osd_state(action="restart", target=osd_id):
+                log.error(f"Unable to restart the OSD : {osd_id}")
+                return 1
+
+        # Sleep for 5 seconds for OSD's to join the cluster
+        time.sleep(5)
```

Comment:
We are already having a blind sleep in `change_osd_state`. Do we still want another one here?